crates/exex/src/exex.rs

```diff
@@ -91,6 +97,50 @@ impl<Node: FullNodeComponents> KakarotRollup<Node> {
         Ok(())
     }
 
+    /// Starts processing chain state notifications with the print transaction hint.
+    pub async fn start_with_transaction_logging(mut self, path: PathBuf) -> eyre::Result<()> {
```

Comment:
I don't understand why you needed to create another function. Why not just adding the hint to the previous one?

Reply:
Just to separate the logic, you prefer having fibonacci + printing hint in the same loop? not a problem for me if so

Reply:
it gives no extra cost to have more hints available in the VM

Reply:
I just fixed it with more than one `start` function in which we pass a vector with several Paths in it so that we can run several programs at once. 

So now in the database we have several traces per block (one for fibonacci and one for printing transaction hashes in the case of our test). 

But really it's not a big deal we don't have to worry too much about this function, the goal was just to be able to run the vm with the hint. In the future, all this will be abstract and we won't need to do it like that anyway.